src/server/mongodb/withMongodb.js

```diff
@@ -1,15 +1,16 @@
 import url from 'url'
 
 import Promise from 'bluebird'
+import config from 'config'
 import { MongoClient } from 'mongodb'
 
 export const connectDB = () => {
   const URL = url.format({
     slashes: true,
-    protocol: 'mongodb',
-    hostname: 'localhost',
-    port: 27017,
-    pathname: 'Interact',
+    protocol: config.get('mongo.protocol'),
+    hostname: config.get('mongo.host'),
+    port: config.get('mongo.port'),
+    pathname: config.get('mongo.db'),
```

Comment:
both of them are supported by node config module with the same performance, but I forget to change `config.get` to `config.mongo` when I implemented this file. :>